### PR TITLE
Selenium Manual UI Testing Framework

### PR DESCRIPTION
**Pull request by GeraldMit**
Addressing #136 by putting basic structure and a sample in place to test roundtrip with non-developer tester on live site manually activated, automated test with Selenium.

### CODE REVIEW

**Comment by davidnixon** [comment]
How do I actually run the tests? I have not used Selenium before. I saw that there is a browser plugin so maybe I should load that and then probably point it at the test suite?

File: tests/selenium/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+
+# Technology
+## Selenium.
+- [Selenium](https://www.selenium.dev)
+
+# For developers
+See the [Getting Started](doc/GETSTARTED.md) document.
+See the [User Guide](doc/5-fifths-user-guide.pdf) document.
+Please read the [CONTRIBUTING guidelines](/CONTRIBUTING.md)
```
Comment (davidnixon) [inline tests/selenium/README.md:42]: these should have a `../..` infront of the links like `../../doc/GETSTARTED.md`

**Comment by GeraldMit** [comment]
yes. the plugin is the easiest way to do a replay for a manual user. I need to add those steps to the readme in the selenium base. directory. I will update those link paths as well.

Selenium github: https://github.com/SeleniumHQ/selenium-ide
Have to use either the Chrome extension or Firefox extension, can get from those browsers' marketplaces or [Chrome extension](https://chrome.google.com/webstore/detail/selenium-ide/mooikfkahbdckldjjndioackbalphokd)
[Firefox Extension](https://addons.mozilla.org/en-GB/firefox/addon/selenium-ide/)

I will setup to run automated tests when I get a chance...  the browser engines can run headless in selenium using the webdriver.

**Comment by krook** [comment]
Is this still something that should be merged? Or are there updates needed @davidnixon and @GeraldMit?

File: tests/selenium/README.md (hunk 1 above)
Comment (krook) [reply tests/selenium/README.md:42]: @GeraldMit did you see @davidnixon's comment above?

**Comment by krook** [comment]
Bump @davidnixon and @GeraldMit?

File: tests/selenium/README.md (hunk 1 above)
Comment (GeraldMit) [reply tests/selenium/README.md:42]: yes, haven't had a chance to get back to it to update

**Comment by GeraldMit** [comment]
> Is this still something that should be merged? Or are there updates needed @davidnixon and @GeraldMit?

yes still needs those updates David pointed out

**Review by krook — APPROVED** [review]

**Comment by davidnixon** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/29862df8-a9ad-4d66-9e9f-fb7dbf74687b?env_id=ibm:yp:us-south "five-fifths-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/73b7a52e-0040-410e-9226-2e9b8435c37a "five-fifths-voter-pipeline") deployed [Five-Fifths-Voter](https://github.com/Call-for-Code-for-Racial-Justice/Five-Fifths-Voter/releases/tag/deploy-ibm-yp-us-south-cfc-team-external-apps-cfc-fivefifths-space-20210809-013010 "deploy-ibm-yp-us-south-cfc-team-external-apps-cfc-fivefifths-space-20210809-013010") to [cfc-fivefifths-space](https://console.bluemix.net/apps/8c251f3c-ac6d-42c1-b64c-cf383c9925bb?env_id=ibm:yp:us-south "ibm:yp:us-south:cfc-team-external-apps:cfc-fivefifths-space"), including commit 7ff075abad852622190574c86f3e85fefd08a96f

**Pull request merged**